tools/bot/server/bot/xrc20.go

```diff
@@ -102,8 +101,8 @@ func (s *Xrc20) checkAndAlert(hs string) {
 	}
 }
 func (s *Xrc20) transfer(pri crypto.PrivateKey) (txhash string, err error) {
-	addr, err := address.FromBytes(pri.PublicKey().Hash())
-	if err != nil {
+	addr := pri.PublicKey().Address()
+	if addr == nil {
 		return
 	}
```

Comment:
err will be nil if not specified in return statement

Reply:
done